packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -159,8 +159,7 @@ export default defineRenderHandler(async (event) => {
   // Render 103 Early Hints
   if (!isRenderingPayload && !process.env.prerender && event.res.socket) {
```

Comment:
It should be checked in h3 already..

```suggestion
  if (!isRenderingPayload && !process.env.prerender) {
```